build/configs/imxrt1050-evk/nxp_demo/defconfig

```diff
@@ -1115,7 +1116,24 @@ CONFIG_EXAMPLES_NXP_DEMO=y
 # CONFIG_EXAMPLES_SMART_TEST is not set
 # CONFIG_EXAMPLES_SPEECH_DETECTOR_TEST is not set
 # CONFIG_EXAMPLES_ST_THINGS is not set
-# CONFIG_EXAMPLES_TESTCASE is not set
+CONFIG_EXAMPLES_TESTCASE=y
+# CONFIG_DISABLE_MANUAL_TESTCASE is not set
+# CONFIG_EXAMPLES_TESTCASE_DRIVERS is not set
+CONFIG_EXAMPLES_TESTCASE_FILESYSTEM=y
```

Comment:
I think its better to create a new defconfig for tc or rename this defconfig.

Reply:
Okay, I'll update the config in the next PR.

Reply:
@chanijjani , board is not booting with nxp_demo config after applying the patch. Can you please confirm.

Reply:
Plz change CONFIG_USER_ENTRYPOINT="nxp_demo_sample_main"  as  CONFIG_USER_ENTRYPOINT="tc_main" and execute again.

Reply:
@yashwanth686007  I addded a new defconfig as imxrt1050-evk/tc. Please try this..

Reply:
@chanijjani , facing compile errors during build.
/home/vulapu/Work/latest/TizenRT/os/../build/output/libraries/libarch.a(imxrt_net.o): In function `up_netinitialize':
/home/vulapu/Work/latest/TizenRT/os/arch/arm/src/chip/imxrt_net.c:66: undefined reference to `rtk_drv_initialize'
/home/vulapu/Work/latest/TizenRT/os/arch/arm/src/chip/imxrt_net.c:71: undefined reference to `lwnl80211_register'
/home/vulapu/Work/latest/TizenRT/os/../build/output/libraries/libdrivers.a(wifi_interactive_mode.o): In function `app_scan_result_handler':
/home/vulapu/Work/latest/TizenRT/os/drivers/wireless/realtek/wifi_util_interface/wifi_interactive_mode.c:917: undefined reference to `wifi_scan_result_callback'

Reply:
I uploaded this as a new PR #3190.
Check it please instead of this one.